admin/sql/create_tables.sql

```diff
@@ -12,6 +12,7 @@ CREATE TABLE "user" (
 );
 ALTER TABLE "user" ADD CONSTRAINT user_musicbrainz_id_key UNIQUE (musicbrainz_id);
 ALTER TABLE "user" ADD CONSTRAINT user_musicbrainz_row_id_key UNIQUE (musicbrainz_row_id);
+ALTER TABLE "user" ADD COLUMN user_login_id VARCHAR;
```

Comment:
this column should be added inside the create table function call instead of creating the table and then altering it.

Reply:
Also, it should be `NOT NULL`, I guess.

Reply:
Also, If the only thing we're filling this column with is UUIDs, we should use the postgres uuid data type instead. (https://www.postgresql.org/docs/9.6/datatype-uuid.html)

This would make it more efficient. And you could set a default value for all users as well.

Reply:
"uuid-ossp" extension is not being created due to some missing files. I will try to fix it and then would use UUID and postgres functions to generate user_login_id.

Reply:
I've fixed the problem in #479. Thanks for the bug report! As soon as it gets merged, you can rebase this PR and everything should work fine. In the meanwhile, you can just install the extension in your "listenbrainz" db and continue.

Reply:
merged now. =)